openslides/users/views.py

```diff
@@ -93,10 +93,15 @@ def update(self, request, *args, **kwargs):
             # The user does not have all permissions so he may only update himself.
             if str(request.user.pk) != self.kwargs['pk']:
                 self.permission_denied(request)
+
+            # This is a hack to make request.data mutable. Otherwise fields can not be deleted.
+            request.data._mutable = True
             # Remove fields that the user is not allowed to change.
             # The list() is required because we want to use del inside the loop.
             for key in list(request.data.keys()):
                 if key not in ('username', 'about_me'):
+                    # request.data is an immutable QueryDict. I can not see the
```

Comment:
Please remove this comment here or - at your choice - add this comment also to all other places where we delete something in the request.

Reply:
done